backend/inspirehep/records/cli.py

```diff
@@ -178,3 +182,49 @@ def _finished_tasks_count():
         LOGGER.warning(f"Got {len(failures)} failures during the updating process")
         for failure in failures:
             LOGGER.warning(failure)
+
+
+@click.group()
+def jobs():
+    """Command for jobs"""
+
+
+@jobs.command(help="Closes jobs with deadline before the given date.")
+@click.option(
+    "--deadline-before-date",
+    help="Date with the format YYYY-MM-DD. All jobs with deadline date before this date "
+    "will be closed.",
+    type=str,
+    default=str(datetime.datetime.utcnow().date()),
+    show_default=True,
+)
+@with_appcontext
+def close_before(deadline_before_date):
+    datetime.datetime.strptime(deadline_before_date, "%Y-%m-%d")
+
+    before_deadline_date = (
+        type_coerce(RecordMetadata.json, JSONB)["deadline_date"].astext.cast(DateTime)
```

Comment:
maybe assign `type_coerce(RecordMetadata.json, JSONB)` to a variable at the beginning? you're using it multiple times.